app/main.py

```diff
@@ -1 +1,54 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.learn_skill("JavaScript")
+        self.learn_skill("HTML")
+        self.learn_skill("CSS")
```

Comment:
Instead of calling `learn_skill` several times, you could extend the `self.skills` directly, e.g.
```suggestion
        self.skills.extend(["JavaScript", "HTML", "CSS"])
```
Make this change in all classes.

Reply:
what about pull request "knights_fighting"???

Reply:
ok, why do we need a function learn_skill?